DNN Platform/DotNetNuke.Web.Deprecated/UI/WebControls/PropertyEditorControls/DateEditControl.cs

```diff
@@ -199,18 +199,8 @@ public override string ID
 				base.ID = value;
 			}
 		}
-
-	    public override string EditControlClientId
```

Comment:
I'll add this back, it was removed without my knowledge. Time gap between the start of this and finalizing the commit.